pygetty/assets.py

```diff
@@ -45,3 +46,43 @@ def individual_asset(
     res.raise_for_status()
 
     return asset_formatters[asset_type](res.json())
+
+
+def multiple_assets(
+    ids,
+    asset_type,
+    detailed=False,
+    fields=set(),
+    query_params={},
+    auth_token_manager=None,
+    api_key=None,
+    client_secret=None,
+):
+    assert asset_type in asset_formatters
+
+    auth_token_manager = flex_auth(
+        auth_token_manager=auth_token_manager,
+        api_key=api_key,
+        client_secret=client_secret,
+    )
+    params = deepcopy(query_params)
+    params['ids'] = ','.join(map(str, ids))
```

Comment:
Isn't it generally recommended (or at least the most readable) to use a comprehension over `map`?

`','.join(ids)` should suffice anyway - `join` always returns a string (which will auto-cast the ideas). Failing that, `','.join(str(id) for id in ids)` I guess?

Reply:
@klardotsh for simple methods I tend to think `map` is more readable
However, good to note that `join` demands string arguments:
```
TypeError: sequence item 0: expected str instance, int found
```